starboard/shared/modular/starboard_layer_posix_pthread_abi_wrappers.cc

```diff
@@ -454,6 +454,37 @@ int __abi_wrap_pthread_mutexattr_destroy(musl_pthread_mutexattr_t* attr) {
   return errno_to_musl_errno(ret);
 }
 
+int __abi_wrap_pthread_mutexattr_gettype(
+    const musl_pthread_mutexattr_t* __restrict attr,
+    int* __restrict type) {
+  int native_type;
+  const pthread_mutexattr_t* tmp = nullptr;
+  if (attr) {
+    tmp = CONST_PTHREAD_INTERNAL_MUTEX_ATTR(attr);
+  }
+  const int ret = pthread_mutexattr_gettype(tmp, &native_type);
+  if (ret != 0) {
+    return errno_to_musl_errno(ret);
+  }
+
+  int musl_type;
```

Comment:
nit: Do we need this variable or can we directly update `*type`?
Same in l.524 with `musl_pshared_val`.